controlplane/src/core/services/Keycloak.ts

```diff
@@ -269,27 +269,10 @@ export default class Keycloak {
     });
   }
 
-  public async deleteOrganizationGroup({ realm, organizationSlug }: { realm?: string; organizationSlug: string }) {
-    const orgGroups = await this.client.groups.find({
-      search: organizationSlug,
-      realm: realm || this.realm,
-      briefRepresentation: true,
-      max: 1,
-    });
-
-    if (orgGroups.length === 0) {
-      return;
-    }
-
-    const orgGroup = orgGroups.find((group) => group.name === organizationSlug);
-
-    if (!orgGroup) {
-      return;
-    }
-
-    await this.client.groups.del({
-      id: orgGroup.id!,
+  public deleteGroupById({ realm, groupId }: { realm?: string; groupId: string }) {
+    return this.client.groups.del({
```

Comment:
will this throw if the group doesnt exist?

Reply:
Yes it does throws

Reply:
Should we have a try catch and return a meaningful error?

Reply:
We can log here, so we find out that it errored out